From b40e5b2eb0c650b2bc550cfc06a9e899e88078de Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:48:17 -0400
Subject: [kernel] perf: Document the new transaction sample type

Message-id: <1408953210-26343-10-git-send-email-jolsa@redhat.com>
Patchwork-id: 88728
O-Subject: [PATCH RHEL7.1 BZ1133083 009/322] perf: Document the new transaction sample type
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 189b84fb54490ae24111124346a8e63f8e019385
Author: Vince Weaver <vince@deater.net>
Date: Fri Dec 13 15:52:25 2013 -0500

description
===========
Commit fdfbbd07e91f8fe3871 ("perf: Add generic transaction flags")
added support for PERF_SAMPLE_TRANSACTION but forgot to add documentation
for the sample type to include/uapi/linux/perf_event.h
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/uapi/linux/perf_event.h b/include/uapi/linux/perf_event.h
index 35a5426..83c1b36 100644
--- a/include/uapi/linux/perf_event.h
+++ b/include/uapi/linux/perf_event.h
@@ -679,6 +679,7 @@ enum perf_event_type {
   *
   * { u64   weight;   } && PERF_SAMPLE_WEIGHT
   * { u64   data_src; } && PERF_SAMPLE_DATA_SRC
+  * { u64   transaction; } && PERF_SAMPLE_TRANSACTION
   * };
   */
  PERF_RECORD_SAMPLE   = 9,
-- 
1.7.1